campaignion_giftaid/campaignion_giftaid.info

```diff
@@ -4,6 +4,7 @@ core = 7.x
 php = 7.0
 
 dependencies[] = campaignion:campaignion_donation
+dependencies[] = campaignion:campaignion_donation_amount
```

Comment:
Is the explicit dependency on campaignion_donation still needed now?

Reply:
Nope … and neither does campaignion_donation_amount really depend on campaignion_donation. I removed both dependencies.